client/my-sites/current-site/index.jsx

```diff
@@ -156,11 +156,11 @@ module.exports = React.createClass( {
 		if ( ! this.props.sites.initialized ) {
 			return (
 				<Card className="current-site is-loading">
-					<div className="site">
 					{ hasOneSite
 						? this.addNewWordPressButton()
-						: <span className="current-site__switch-sites" />
+						: <span className="current-site__switch-sites">&nbsp;</span>
```

Comment:
Wouldn't react render this as plain-text `&nbsp;` ? Should this be a space charater ` ` instead?